api/src/main/java/org/pmiops/workbench/monitoring/MonitoringServiceImpl.java

```diff
@@ -42,13 +43,13 @@ private void initStatsConfigurationIdempotent() {
   }
 
   private void registerSignals() {
-    Arrays.stream(MonitoringViews.values())
-        .map(MonitoringViews::toOpenCensusView)
+    Arrays.stream(GaugeMetric.values())
+        .map(OpenCensusView::toView)
         .forEach(viewManager::registerView);
   }
 
   @Override
-  public void recordValue(OpenCensusStatsViewInfo viewInfo, Number value) {
+  public void recordValue(OpenCensusView viewInfo, Number value) {
```

Comment:
I'm a confused by the method override here. Why is a default method implementation provided in the interface if the only concrete class overrides it? I suspect this was lost in refactoring and should be removed.